webpack/assets/javascripts/react_app/components/HostsIndex/Columns/core.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { translate as __ } from '../../../common/I18n';
+import RelativeDateTime from '../../common/dates/RelativeDateTime';
+import forceSingleton from '../../../common/forceSingleton';
+
+const coreHostsIndexColumns = [
+  {
+    columnName: 'name',
+    title: __('Name'),
+    wrapper: ({ name }) => <Link to={`hosts/${name}`}>{name}</Link>,
+    isSorted: true,
+    weight: 0,
+  },
+  {
+    columnName: 'hostgroup',
+    title: __('Host group'),
+    wrapper: hostDetails => hostDetails?.hostgroup_name,
+    isSorted: true,
+    weight: 100,
+  },
+  {
+    columnName: 'os_title',
+    title: __('OS'),
+    wrapper: hostDetails => hostDetails?.operatingsystem_name,
+    isSorted: true,
+    weight: 200,
+  },
+  {
+    columnName: 'owner',
+    title: __('Owner'),
+    wrapper: hostDetails => hostDetails?.owner_name,
```

Comment:
I'm not sure if it's good or not, but owners can be users or groups. In some places we've used an icon to differentiate. Something to consider. I'm not sure today you **can** distinguish between the two in the REST API, but that could be a good reason to enhance the API.

Reply:
Example:
![image](https://github.com/theforeman/foreman/assets/14796566/44e62c7c-5f1f-4a51-8cd0-b961a8644b99)